backend/src/main/resources/db/migration/V31_RegistrationStatus.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE "registration"
+ADD COLUMN registration_status varchar(32);
+
+UPDATE "registration"
+SET registration_status = 'REGISTERED'
+WHERE wait_list = FALSE;
+
+UPDATE "registration"
+SET registration_status = 'WAITLIST'
+WHERE wait_list = TRUE;
+
+ALTER TABLE "registration"
+    ALTER COLUMN registration_status
+        SET NOT NULL;
+
+ALTER TABLE "registration"
+DROP COLUMN wait_list;
```

Comment:
1. Denne må hete `V31__RegistrationStatus.sql`, altså med to understreker.
2. Mangler kolonnene `reason` og `deregistration_date`.